frontend/src/pages/PipelineDetails.test.tsx

```diff
@@ -315,6 +317,8 @@ describe('PipelineDetails', () => {
   });
 
   it('shows no graph error banner when failing to parse graph', async () => {
+    // TODO: Figure out why v1 argo template instance cannot get recognized as plain object with key-value pair in test.
```

Comment:
I think this Todo should be fixed in this PR. What do you think? It might be a bug

Reply:
You are right! Thanks for pointing it out, I found and fixed this issue in test